pkg/controllers/dynakube/istio/reconciler_test.go

```diff
@@ -450,6 +508,57 @@ func TestReconcileActiveGateCommunicationHosts(t *testing.T) {
 	})
 }
 
+func TestParseCodeModulesImageURL(t *testing.T) {
+	tests := []struct {
+		input           string
+		output          string
+		parsedCorreclty bool
+	}{
+		{
+			input:           "some.url.com/test",
+			output:          "https://some.url.com/test",
+			parsedCorreclty: true,
+		},
+		{
+			input:           "http://some.url.com/test",
+			output:          "http://some.url.com/test",
+			parsedCorreclty: true,
+		},
+		{
+			input:           "https://some.url.com/test",
+			output:          "https://some.url.com/test",
+			parsedCorreclty: true,
+		},
+		{
+			input:           ":example.com/test",
+			output:          "",
+			parsedCorreclty: false,
+		},
+		{
+			input:           "//some.url.com/test",
+			output:          "https://some.url.com/test",
+			parsedCorreclty: true,
+		},
+	}
```

Comment:
be aware that the URL you are parsing is an image url (as you pass in the `dk.Status.CodeModules.ImageID` directly), so it will have a `:tag` as a postfix, be sure to test that scenario

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/3343/commits/619e50a181dc45bb873290e4cd0f59372eadecdc